bot/cogs/core/mobs.py

```diff
@@ -288,7 +289,11 @@ async def _spawn_event(self, ctx: Ctx):
                     await ctx.send_embed(random.choice(mob.finishers))
                     break
                 # send mob attack
-                await ctx.send_embed(random.choice(mob.attacks))
+                mob_attack_text = random.choice(mob.attacks)
+                if "{}" in mob_attack_text:
+                    mob_attack_text = mob_attack_text.format(datetime.now(tz=timezone.utc).year)
```

Comment:
```suggestion
                if "{atk_year}" in mob_attack_text:
                    mob_attack_text = mob_attack_text.format(atk_year=datetime.now(tz=timezone.utc).year)
```

We could do this to facilitate future placeholders 👀